pkg/dao/templates.go

```diff
@@ -589,6 +592,62 @@ func (t templateDaoImpl) DeleteTemplateSnapshot(ctx context.Context, snapshotUUI
 	return nil
 }
 
+func (t templateDaoImpl) GetRepositoryConfigurationFiles(ctx context.Context, orgID, templateUUID string) (string, error) {
+	sDao := snapshotDaoImpl(t)
+	rcDao := repositoryConfigDaoImpl{db: t.db}
+
+	template, err := t.Fetch(ctx, orgID, templateUUID, false)
+	if err != nil {
+		return "", err
+	}
+
+	pc := t.pulpClient
+	contentPath, err := pc.GetContentPath(ctx)
+	if err != nil {
+		return "", err
+	}
+
+	var templateRepoConfigFiles strings.Builder
+	for _, snap := range template.Snapshots {
+		repoConfig, err := rcDao.fetchRepoConfig(ctx, orgID, snap.RepositoryUUID, true)
+		if err != nil {
+			return "", err
+		}
+
+		repoConfigFile, err := sDao.GetRepositoryConfigurationFile(ctx, orgID, snap.UUID, false)
+		if err != nil {
+			return "", err
+		}
+
+		var contentURL string
+		domain := strings.Split(snap.RepositoryPath, "/")[0]
+		parsedRepoURL, err := url.Parse(repoConfig.Repository.URL)
+		if err != nil {
+			return "", err
+		}
+		path := parsedRepoURL.Path
+		if domain == config.RedHatDomainName {
+			contentURL = contentPath + domain + "/templates/" + templateUUID + path
```

Comment:
I'm seeing the path for red hat repo URLs as `.../templates/677b42dd-f08b-4512-bc46-af9fa4bcca14/2609d309-37ff-45d8-a39a-7aae3d4b9e14`. I think we probably need to check the org here instead of the domain

Reply:
or I might just need to reset my database

Reply:
hmm i haven't reproduced yet but yep should be checking org here, let me know if that helps?

Reply:
it does, but also my database was using an old domain name